generator/ServiceModels/s3/s3.customizations.json

```diff
@@ -381,6 +388,13 @@
         },
         "StorageClass":{
             "renameShape": "S3StorageClass"
+        },
+        "Payer":{
+            "renameShape": "string"
         }
+
+    },
+    "treatEnumsAsEnums":{
```

Comment:
Instead of making up the new term `treatEnumsAsEnums` which by itself sounds weird because if set it false then what does it treat enums if not enums. I suggest reusing the existing term and possible add an `Override` prefix. For example have a customization called `OverrideTreatEnumsAsString` and then add `ObjectAttributesList` with a value `false`.